src/dodal/beamlines/b07.py

```diff
@@ -0,0 +1,21 @@
+from dodal.common.beamlines.beamline_utils import (
+    device_factory,
+)
+from dodal.common.beamlines.beamline_utils import set_beamline as set_utils_beamline
+from dodal.devices.electron_analyser.specs_analyser_io import (
+    SpecsAnalyserDriverIO,
+)
+from dodal.log import set_beamline as set_log_beamline
+from dodal.utils import BeamlinePrefix, get_beamline_name
+
+BL = get_beamline_name("b07")
+PREFIX = BeamlinePrefix(BL, suffix="B")
+set_log_beamline(BL)
+set_utils_beamline(BL)
+
+
+@device_factory()
+def analyser_driver() -> SpecsAnalyserDriverIO:
+    return SpecsAnalyserDriverIO(
+        name="analyser_driver", prefix=f"{PREFIX.beamline_prefix}-EA-DET-01:CAM:"
+    )
```

Comment:
nit: if you don't provide a name, the name of the function is used, I want to encourage not explicitly naming the device so the behaviour is consistent